test/activities/CategoriesEntity.js

```diff
@@ -138,7 +138,7 @@ describe('CategoriesEntity', () => {
 
 	describe('Saves', () => {
 		it('saves categoryId', async() => {
-			await fetchMock.patchOnce('https://afe99802-9130-4320-a770-8d138b941e74.assignments.api.proddev.d2l/6606/folders/13', editableEntity);
+			await fetchMock.patchOnce('https://afe99802-9130-4320-a770-8d138b941e74.activities.api.proddev.d2l/6606/folders/13', editableEntity);
```

Comment:
This url seems strange to me `https://afe99802-9130-4320-a770-8d138b941e74.activities.api.proddev.d2l/6606/folders/13`. Does it actually exist? I.e. an `activities` api ending in `folders/<id>`?

Reply:
Nope it doesn't exist...we would actually patch:

- `https://afe99802-9130-4320-a770-8d138b941e74.assignments.api.proddev.d2l/6606/folders/13`
- `https://afe99802-9130-4320-a770-8d138b941e74.quizzes.api.proddev.d2l/6606/quizzes/13`

but I guess I was trying to make it more generic so the next person who came along didn't think it was specific to a particular tool. 

I'll put up a new PR to improve this.